### DISCUSSION TITLE
Authorization error when running in gitlab

### DISCUSSION THREAD

**Discussion by lucy7099** [percy/cli#548] [Support] [answered]
[](url)hi when running my percy tests i get this error on gitlab when running the percy snapshot file in my pipeline.
i am already passing the PERCY_TOKEN in my gitlab-ci file and set it up on gitlab. any idea what i am missing?

when i am running locally it all works fine. 

<img width="808" alt="Screenshot 2021-09-08 at 16 37 32" src="https://user-images.githubusercontent.com/89091183/132540595-c290b116-60f0-4e4e-a644-5270e54d6b8a.png">

****

this is my snapshot.yml file

> - name: Homepage Snapshot
>   url: "http://localhost:3000/search?...."
>   waitForTimeout: 2000
>   waitForSelector: .ticket-panel__container

**Comment by Robdel12** [comment, marked as answer]
Hey @lucy7099! The error seems to suggest there's an encoding issue with what `PERCY_TOKEN` is set to. It could be something like smart quotes over regular quotes causing that issue

**Reply by lucy7099** [reply]
yes i had a line break by mistake in gitblab where i set the token value. thanks
